kitchen.dokken.yml

```diff
@@ -40,27 +40,25 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
 
-  - name: centos-6
-    driver:
-      image: dokken/centos-6
-      pid_one_command: /sbin/init
-
   - name: centos-7
     driver:
       image: dokken/centos-7
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: fedora-29
+  - name: centos-8
     driver:
-      image: dokken/fedora-29
+      image: dokken/centos-8
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: ubuntu-16.04
+  - name: fedora-30
```

Comment:
it's probably easier to just change this over to fedora-latest and never have to update it

Reply:
good call

Reply:
changed in latest commit